docs/user/subprojects.rst

```diff
@@ -1,46 +1,74 @@
 Subprojects: host multiple projects on a single domain
-=====================================================
+======================================================
 
-Projects can be configured in a nested manner, by configuring a project as a
-*subproject* of another project. This allows for documentation projects to share
-a search index and a namespace or custom domain, but still be maintained
-independently.
+In this article, you can learn more about how several documentation projects can be combined and presented to the reader on the same website.
+
+Read the Docs projects can be configured to make other *subprojects* available on the website of the *main project*.
+This allows for documentation projects to share a search index and a namespace or custom domain,
+but still be maintained independently.
 
 This is useful for:
 
 * Organizations that need all their projects visible in one documentation portal or landing page
 * Projects that document and release several packages or extensions
 * Organizations or projects that want to have a common search function for several sets of documentation
 
-For example, a parent project, ``foo`` is set up with a subproject, ``bar``. The
-documentation for ``foo`` will be available at:
+For example, a parent project, ``example-project`` is set up with a subproject, ``plugin``. The
+documentation for ``example-project`` will be available at:
 
-https://foo.readthedocs.io/en/latest/
+https://example-project.readthedocs.io/en/latest/
 
-The documentation for ``bar`` will be available under this same path:
+The documentation for ``plugin`` will be available under this same path:
 
-https://foo.readthedocs.io/projects/bar/en/latest/
+https://example-project.readthedocs.io/projects/plugin/en/latest/
 
 .. seealso::
 
    :doc:`/guides/subprojects`
 
-
 Sharing a custom domain
 -----------------------
 
-Projects and subprojects can also be used to share a custom domain with a number
-of projects. To configure this, one project should be established as the parent
-project. This project will be configured with a custom domain. Projects can then
-be added as subprojects to this parent project.
+Projects and subprojects can be used to share a custom domain.
+To configure this, one project should be established as the main project.
+This project will be configured with a custom domain.
+Other projects are then added as subprojects to the main project.
 
 If the example project ``foo`` was set up with a custom domain,
 ``docs.example.com``, the URLs for projects ``foo`` and ``bar`` would
-respectively be at: 
+respectively be at:
 
 * `foo`: https://docs.example.com/en/latest/
 * `bar`: https://docs.example.com/projects/bar/en/latest/
 
+.. note::
+
+   The terms "parent" and "child" are also used,
+   where the main project is the parent and all subprojects are a child.
+   You can **not** have a child of a child,
+   meaning that you cannot add a subproject to a subproject.
```

Comment:
This is great, but we should probably stick to a single language if possible. 👍

Reply:
The reason for the note was that we actually have "child" in the form field.

Should I cross a line and rename the form label directly in this PR? :see_no_evil:

Reply:
Ah, okay, no, but I can open a separate PR and remove the whole note admonition